src/Service/CMakeLists.txt

```diff
@@ -13,18 +13,15 @@ target_sources(OrbitServiceLib PRIVATE
         OrbitGrpcServer.cpp
         OrbitGrpcServer.h
         OrbitService.cpp
-        OrbitService.h
-        ProducerSideServer.cpp
-        ProducerSideServer.h
-        ProducerSideServiceImpl.cpp
-        ProducerSideServiceImpl.h)
+        OrbitService.h)
 
 target_include_directories(OrbitServiceLib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
 target_link_libraries(OrbitServiceLib PUBLIC
         GrpcProtos
         OrbitVersion
         ProducerSideChannel
+        ProducerSideService
```

Comment:
Can you also create a `BUILD` file and add it as a dependency in `src/Service/BUILD` such that the internal build works?

Reply:
Done! Thanks for pointing this out.